data/outreachy.yml

```diff
@@ -0,0 +1,127 @@
+          provide tooling on top of the messages.
+        mentee: "Muskan Garg"
+        blog: "https://medium.com/@muskangarg994/outreachy-101-d285507c54f7"
+        mentors:
+         - Florian Angeletti
+  - name: Summer 2021
+    projects:
+      - title: "Create opam package search"
+        description: >
+          opam is the source-based package manager for OCaml code. This project comprises of writing a new web
+          client for rendering output from the opam package database. There is a JSON endpoint on opam.ocaml.org
+          which provides information about packages which would provide metadata about the packages.  We can extend
+          this JSON metadata to include all the opam packages (not just the top 10) and use that to power a search
+          frontend for the website. This may include presenting the data as a GraphQL endpoint with the frontend
+          querying that endpoint using GraphQL.
+        mentee: "Odinaka Joy"
+        blog: "https://www.dinakajoy.com"
+        mentors:
+         - Sonja Heinze
+         - Patrick Ferris
+      - title: "Improve the ocaml.org website"
+        description: >
+          OCaml.org is the main website for OCaml, a functional, typed, high-level programming language. This project
+          revolves around improving the website on multiple different fronts including: layout, accessibility and content.
+        mentee: "Diksha Gupta"
+        blog: "https://dikshagupta99.wordpress.com/blog/"
+        mentors:
+         - Isabella Leandersson
+         - Patrick Ferris
+         - Gargi Sharma
+      - title: "Improve the ocaml.org website"
+        description: >
+          OCaml.org is the main website for OCaml, a functional, typed, high-level programming language. This project
+          revolves around improving the website on multiple different fronts including: layout, accessibility and content.
+        mentee: "Shreya kumari Gupta"
+        blog: "https://shreyaswikriti693235797.wordpress.com/blog/"
+        mentors:
+         - Isabella Leandersson
+         - Anil Madhavapeddy
+         - Patrick Ferris
+         - Gargi Sharma
+  - name: Winter 2021
+    projects:
+      - title: "Build a monitoring dashboard for OCaml.org"
+        description: ""
+        mentee: "Jiae Kam"
+        blog: "https://jiaek.wordpress.com"
+        mentors:
+         - Thibaut Mattio
+      - title: "Improve the OCaml meta-programming ecosystem"
+        description: ""
+        mentee: "Aya Sharaf"
+        blog: "https://outreachyxocaml.wordpress.com"
+        mentors:
+         - Shon Feder
+         - Sonja Heinze
+      - title: "Support .eml files in OCaml's VSCode extension"
+        description: ""
+        mentee: "Sayo Bamigbade"
+        blog: "https://saysayoblog.wordpress.com"
+        mentors:
+         - Thibaut Mattio
+         - Gargi Sharma
+  - name: Summer 2022
+    projects:
+      - title: "Expand OCaml 5.0 Parallel Benchmark suite"
+        description: ""
```

Comment:
OCaml 5.0 will be live soon! It ships with support for shared-memory parallelism and concurrency OCaml has missed all these years. This will be accompanied by a robust set of Multicore libraries useful for parallel programming. The Multicore compiler and libraries are under active development and will continue to evolve as the OCaml ecosystem moves towards Multicore.

For assessing the impact of new features in the OCaml compiler and Multicore libraries, we have a set of sequential and parallel benchmarks present in our benchmark suite. While the sequential benchmarks contain many real-world applications, a wider set of parallel benchmarks would be useful.

This project entails gathering the parallel benchmarks available at various places like https://github.com/ckoparkar/ocaml-benchmarks and making them available in the benchmark suite.

Reply:
Thanks! Added :))